tools/sideloading/install.py

```diff
@@ -0,0 +1,53 @@
+import sys
+import os
+import shutil
+import argparse
+import subprocess
+
+
+argparser = argparse.ArgumentParser( description = 'Panorama sideloading installation script' )
+argparser.add_argument('--expiry', action='store', type=int, default=30, help='Expiry of certificates in days (default: 30 days)')
+argparser.add_argument('--app-src-dir', dest="app_src_dir", action='store', required=True, help="Path to Panorama application's src directory")
+args = argparser.parse_args()
+
+
+def copy_file( src, dst_dir ):
+    
+    if not os.path.isdir(dst_dir):
+        raise ValueError( f"{dst_dir} is not a directory" )
+
+    print( f"Copying {src} to {dst_dir}" )
+
+    filename = os.path.basename(src)
+    dst = os.path.join( dst_dir, filename )
+
+    shutil.copyfile( src, dst )
+
+def delete_file( filename ):
+
+    print( f"Deleting {filename}" )
+    os.unlink( filename )
+
+def install_certs_keys():
+
+    # Generate certs and keys
+    subprocess.run( [ "openssl", "req", "-x509", "-new", "-days", str(args.expiry), "-nodes", "-out", "sideloading_server.cert.pem", "-keyout", "sideloading_server.key.pem", "-subj", "/CN=pan-sideloading-server" ] )

```

Comment:
Note the dependency on OpenSSL in the readme.
Can you check the OpenSSL version to make sure it is supported?
Can you specify the algorithm and key size?